<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>header</class>
 <widget class="QWidget" name="header">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>100</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;

border:0;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(250,250,250);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>0</y>
             <width>291</width>
             <height>76</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter">
             <widget class="QPushButton" name="Name">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:13pt;
color:black;
font-family:Roboto;
border:0;</string>
              </property>
              <property name="text">
               <string>Mohamed Ramdan</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
             <widget class="QPushButton" name="lastSeen">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:11pt;
color:gray;
font-family:Roboto;</string>
              </property>
              <property name="text">
               <string>Last seen</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>18</x>
             <y>8</y>
             <width>60</width>
             <height>60</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../assets.qrc">:/icons/pfp.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="horizontalSpacing">
            <number>16</number>
           </property>
           <item row="0" column="1">
            <widget class="QPushButton" name="Search">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../assets.qrc">
               <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="Dots">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../assets.qrc">
               <normaloff>:/icons/3 dots.png</normaloff>:/icons/3 dots.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="addContact">
             <property name="styleSheet">
              <string notr="true">border-radius: 12px; 
background-color: &quot;#D3E8D2&quot;;
padding: 8px 10px;</string>
             </property>
             <property name="text">
              <string>Add Contact</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../assets.qrc"/>
 </resources>
 <connections/>
</ui>
